public/docs/ts/latest/guide/index.jade

```diff
@@ -9,8 +9,6 @@ figure
   This is a practical guide to Angular for experienced programmers who
```

Comment:
This wording (and all references to "guide") is left over from when BASICS and DEVELOPER GUIDE were all one thing (DEVELOPER GUIDE, I think). I think it predated the cookbook.

I'm not sure how we should refer to this group of docs. Nor am I sure whether this text refers to the whole site or just to stuff under guide/.

@wardbell or @naomiblack, maybe you have ideas?